tests/mock.py

```diff
@@ -70,5 +70,20 @@ class JWPlatformMock(HttpApiMock):
                     "type": "media",
                 }],
             })
+        ),
+        HttpApiMockEndpoint(
+            operation_id='createMedia',
+            match_pattern=b'^POST /v2/sites/(?P<site_id>[A-Za-z0-9]{8}?)/media/',
+            response=lambda _: (200,
+                                {"upload_id": "uploADid",
+                                 "upload_token": "upload_token",
+                                 "upload_link": "http://s3server/upload-link",
+                                 "upload_method": "upload_method"}
```

Comment:
btw the media endpoint won't actually return upload method (dunno if this is just used for testing purposes elsewhere)